docs/conf.py

```diff
@@ -77,7 +77,7 @@
 #
 # This is also used if you do content translation via gettext catalogs.
 # Usually you set "language" from the command line for these cases.
-language = None
+# language = "en"
```

Comment:
Did we mean to comment this line out? Asking because we also changed the value to `"en"`.

Reply:
I agree, it is confusing. "en" is the default, and also appropriate for our project. I'll uncomment.